roles/kustomize_deploy/defaults/main.yml

```diff
@@ -0,0 +1,136 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_kustomize_deploy"
+
+cifmw_kustomize_deploy_destfiles_basedir: >-
+  {{
+    cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data')
+  }}
+
+cifmw_kustomize_deploy_architecture_repo_url: "https://github.com/openstack-k8s-operators/architecture"
+
+cifmw_kustomize_deploy_architecture_repo_dest_dir: >-
+  {{
+      [
+        ansible_user_dir,
+        'src',
+        'github.com',
+        'openstack-k8s-operators',
+        'architecture'
+      ] | path_join
+  }}
+
+cifmw_kustomize_deploy_architecture_repo_version: "HEAD"
+
+cifmw_kustomize_deploy_cp_values_src_file: >-
+  {{
+    [
+      cifmw_kustomize_deploy_destfiles_basedir,
+      'artifacts',
+      'ci_gen_kustomize_values',
```

Comment:
values.yaml are now in sub-directories:
`ci_gen_kustomize_values/(network-values|edpm-values)/values.yaml`
The network-values matches "ctlplane" while edpm-values are for dataplane.